tests/examples/llm/vdb_upload/test_schema_transform_module.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import types
+
+import pytest
+
+import cudf
+
+from _utils import TEST_DIRS
+from _utils import assert_results
+from morpheus.config import Config
+from morpheus.messages import MessageMeta
+from morpheus.pipeline import LinearPipeline
+from morpheus.stages.general.linear_modules_stage import LinearModulesStage
+from morpheus.stages.input.in_memory_source_stage import InMemorySourceStage
+from morpheus.stages.output.compare_dataframe_stage import CompareDataFrameStage
+
+path = os.path.join(TEST_DIRS.examples_dir, 'llm/vdb_upload/')
+sys.path.append(path)
```

Comment:
I would make this a fixutre, and depend on the `restore_sys_path` otherwise other tests will have this in their sys.path, something like:
```
@pytest.fixture(autouse=True)
def example_dir_in_sys_path(restore_sys_path):
    path = os.path.join(TEST_DIRS.examples_dir, 'llm/vdb_upload/')
    sys.path.append(path)